docs/usage.rst

```diff
@@ -125,3 +125,21 @@ Pre-0.7.0, this is how it worked:
 
 * Cloned cookiecutters were not saved locally.
 
+Update a cookiecutter generated project
+---------------------------------------
+
+If you want to update a project generated by a cookiecutter template, give its PATH to
+the -u option of cookiecutter. This project should have a `.cookiecutter.json` file to
```

Comment:
How is the `.cookiecutter.json` file created? Does the user need to do this manually? It would be cool (I think) if running the cookiecutter to create an initial project *also* created this file, so it could be included in the initial commit to the `cookiecutter` branch of the new project.

Reply:
@barrywhart, this is done with a 
`.cookiecutter.json` file in your template with the following content.

```
{{ cookiecutter | jsonify }}
```

Reply:
Great tip, thanks so much!